packages/devextreme/js/__internal/ui/html_editor/m_html_editor.ts

```diff
@@ -140,37 +167,39 @@ class HtmlEditor extends Editor<Properties> {
     super._initTemplates();
   }
 
-  _focusTarget() {
+  _focusTarget(): dxElementWrapper {
     return this._getContent();
   }
 
   _getContent(): dxElementWrapper {
     return this.$element().find(`.${HTML_EDITOR_CONTENT_CLASS}`);
   }
 
-  // @ts-expect-error ts-error
-  _focusInHandler({ relatedTarget }): void {
+  _focusInHandler(e: DxEvent): void {
```

Comment:
```suggestion
  _focusInHandler(e: DxEvent & { relatedTarget?: Element }): void {
```

And we can remove `as`